src/services/timer/service/jobs/index.ts

```diff
@@ -0,0 +1,188 @@
+> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: 'https://auth.globus.org/scopes/524230d7-ea86-4a52-8312-86065a9e0417/timer',
+      path: `/jobs/${job_id}`,
+      method: HTTP_METHODS.GET,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    query?: OpenAPI.operations['read_job_jobs__job_id__get']['parameters']['query'];
+    payload?: never;
+  }
+>;
+
+/**
+ * @see https://timer.automate.globus.org/redoc#tag/jobs/operation/update_job_jobs__job_id__patch
+ */
+export const patch = function (
+  job_id,
+  options,
+  sdkOptions?: SDKOptions,
+): Promise<
+  JSONFetchResponse<
+    OpenAPI.operations['update_job_jobs__job_id__patch']['responses']['200']['content']['application/json']
+  >
+> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: 'https://auth.globus.org/scopes/524230d7-ea86-4a52-8312-86065a9e0417/timer',
+      path: `/jobs/${job_id}`,
+      method: HTTP_METHODS.PATCH,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    payload: OpenAPI.operations['update_job_jobs__job_id__patch']['requestBody']['content']['application/json'];
+  }
+>;
+
+/**
+ * @see https://timer.automate.globus.org/redoc#tag/jobs/operation/pause_job_jobs__job_id__pause_post
+ */
+export const pause = function (
+  job_id,
+  options,
+  sdkOptions?: SDKOptions,
+): Promise<
+  JSONFetchResponse<
+    OpenAPI.operations['pause_job_jobs__job_id__pause_post']['responses']['200']['content']['application/json']
+  >
+> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: 'https://auth.globus.org/scopes/524230d7-ea86-4a52-8312-86065a9e0417/timer',
+      path: `/jobs/${job_id}/pause`,
+      method: HTTP_METHODS.POST,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    payload?: never;
+    body?: never;
+    query?: never;
+  }
+>;
+
+/**
+ * @see https://timer.automate.globus.org/redoc#tag/jobs/operation/resume_job_jobs__job_id__resume_post
+ */
+export const resume = function (
+  job_id,
+  options,
+  sdkOptions?: SDKOptions,
+): Promise<
+  JSONFetchResponse<
+    OpenAPI.operations['resume_job_jobs__job_id__resume_post']['responses']['200']['content']['application/json']
+  >
+> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: 'https://auth.globus.org/scopes/524230d7-ea86-4a52-8312-86065a9e0417/timer',
+      path: `/jobs/${job_id}/resume`,
+      method: HTTP_METHODS.POST,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    payload?: never;
+    body?: never;
+    query?: OpenAPI.operations['resume_job_jobs__job_id__resume_post']['parameters']['query'];
+  }
+>;
+
+/**
+ * @see https://timer.automate.globus.org/redoc#tag/jobs/operation/delete_timer_jobs__timer_id__delete
+ */
+export const remove = function (
+  job_id,
+  options,
+  sdkOptions?: SDKOptions,
```

Comment:
```suggestion
  sdkOptions?,
```

You don't need to type the `sdkOptions?` arguments in these signatures, the `satisfies` should take care of it.